print_list.js

```diff
@@ -0,0 +1,34 @@
+
+class Node {
+  constructor(value, next) {
+    this.value = value;
+    this.next = next;
+  }
+
+  getNext() {
+    return this.next;
+  }
+
+  setNext(next) {
+    this.next = next;
+  }
+
+  getValue() {
+    return this.value;
+  }
+
+  setValue(value) {
+    this.value = value;
+  }
+
+  printList() {
+    let current = this;
+
+    while (current != null) {
+      console.log(current.value);
+      current = current.getNext();
+    }
+  }
+}
+
+export default Node;
```

Comment:
This code looks really great, there are just a few things that I would appreciate some clarity on, thank you!